libethereum/State.cpp

```diff
@@ -228,10 +238,10 @@ unordered_map<Address, u256> State::addresses() const
 {
 #if ETH_FATDB
 	unordered_map<Address, u256> ret;
-	for (auto& i: m_cache)
+	for (auto& i : m_cache)
```

Comment:
so should here be a space or not? Youichi is always fixing this in my  PRs

Reply:
@winsvega I'm following the current CodingStandards.txt.  After this PR is merged, I'll stop doing that.

Reply:
There is not way to do the version without space in clang-format, so we will use space from now on.
Not having this option in clang-format usually means: nobody in the world of C++ cares and formats the code like that.

Reply:
Yeah I never saw anything like that before cpp-etherem